tools/clang/test/HLSLFileCheck/hlsl/types/struct/bitfields.hlsl

```diff
@@ -0,0 +1,137 @@
+// RUN: %dxc /Zi /T vs_6_2 /E main %s | FileCheck %s
+
+RWByteAddressBuffer BufferOut : register(u0);
+
+// The store of p.y into the temporary was not
+// getting extended to 32 bits, so uninitialized
+// bits of the temporary were used.  7366161.
+struct foo {
+  int x : 8;
+  int : 8;
+  int y : 16;
+};
+
+int main(struct foo p : IN0, int x : IN1) : OUT {
+  // CHECK: %1 = {{.*}}loadInput{{.*}} i32 0, i32 0, {{.*}}
+  // CHECK:      ashr i32 %1
+  // %[[val:.*]] = {{.*}}loadInput{{.*}} i32 0, i32 0, {{.*}}
```

Comment:
Above, your CHECK is strict about the value label, which could cause failures depending on whether this is run on a Release or Debug version of the compiler.  Here you have a more general version, but it's not used.  Was this intended to replace the above CHECK: contents?

Reply:
Yes, that was a temporary debugging change I'd forgotten about.  I've added the check to exclude the quoted source file as well.